Connection.php

```diff
@@ -287,12 +287,21 @@ protected function _run(HoaSocket\Node $node)
                             break;
                         }
 
-                        $this->getListener()->fire(
-                            'message',
-                            new Event\Bucket([
-                                'message' => $frame['message']
-                            ])
-                        );
+                        try {
```

Comment:
Do you think it would be bad to catch with `HoaException\Exception` instead of  `HoaException\Group`? This is larger and less “breakable”.